src/chart_types/heatmap/state/selectors/tooltip.ts

```diff
@@ -48,9 +48,37 @@ export const getTooltipInfoSelector = createCachedSelector(
       pickedShapes
         .filter(({ visible }) => visible)
         .forEach((shape) => {
-          const xValueLabel = config.xAxisLabel.formatter(shape.datum.x);
+          // X-axis value
           tooltipInfo.values.push({
-            label: `${shape.datum.y} - ${xValueLabel}`,
+            label: config.xAxisLabel.name,
+            color: RGBtoString(shape.fill.color),
+            isHighlighted: false,
+            isVisible: true,
+            seriesIdentifier: {
+              specId: spec.id,
+              key: spec.id,
+            },
+            value: `${shape.datum.x}`,
+            formattedValue: config.xAxisLabel.formatter(shape.datum.x),
+          });
+
+          // Y-axis value
+          tooltipInfo.values.push({
+            label: config.yAxisLabel.name,
+            color: RGBtoString(shape.fill.color),
+            isHighlighted: false,
+            isVisible: true,
+            seriesIdentifier: {
+              specId: spec.id,
+              key: spec.id,
+            },
+            value: `${shape.datum.y}`,
+            formattedValue: config.yAxisLabel.formatter(shape.datum.y),
+          });
+
+          // Cell value
+          tooltipInfo.values.push({
+            label: '',
```

Comment:
Here, on XYAxis charts we use the `spec.name` or the `spec.id` as fallback
```suggestion
            label: label: spec.name ?? spec.id,
```
You can add an optional `name` parameter on the Heatmap spec to use in this case. an empty `name` can be used if the user don't want the default parameter.

Reply:
Fixed in https://github.com/elastic/elastic-charts/pull/847/commits/b8081acaaaad1740022c85d4912c52d0ea80085e